### ISSUE TITLE
In 3.0beta, `org-gtd-mode` causes tasks to show up multiple times in `org-agenda`

### ISSUE THREAD

**Issue by Trevoke** [Trevoke/org-gtd.el#142]

**Comment by Trevoke** [comment]
@maikol-solis this should now be fixed on on the org-gtd-3 branch as well.

**Comment by maikol-solis** [comment]
Hi! I tried on my config, and the change in this commit does not work for me: https://github.com/Trevoke/org-gtd.el/blob/d30eda351d780b94bd07ca1049c61cabfbefbdae/org-gtd-core.el#L216-L220

The issue comes because I had `org-agenda-files` and `org-gtd-directory` reading the same files, therefore the duplication. 

My workaround was to do `(setq org-agenda-files nil)` and this sanitized the agenda view with one line per task as expected.

**Comment by Trevoke** [comment]
Ah! You're right, this is why I shouldn't do work at 10pm in the evening. I
didn't actually fix the root cause. I have to find the place where the
org-agenda logic actually expands this.

**Comment by maikol-solis** [comment]
Yes, sure. No problem. Glad that at least the problem was identified.

**Comment by Trevoke** [comment]
Okay this should do it.

**Comment by maikol-solis** [comment]
With the custom function for the agenda views, it should resolve the problem definitely. 

Thanks.
